README.md

```diff
@@ -186,6 +186,24 @@ to `await`!
 `type` will click the element before typing. To disable this, set the
 `skipClick` option to `true`.
 
+
+An example of using `{space}` with a button:
```

Comment:
Every other example cames after the explanation, what if you merge this example along with this [current example](https://github.com/testing-library/user-event/pull/538/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R250) leaving two tests wrapped in a `describe`?

Example:
```jsx
describe('Special characters examples', () => {
  test('delete characters within the selectedRange', () => {
    // ...
  })
  test('emit click on button for {space}', () => {
    // ...
  })
})
```

Reply:
That makes sense to me.